src/utils/TemplateMaker.js

```diff
@@ -0,0 +1,45 @@
+const { UPSIDE_SYMBOL, DOWNSIDE_SYMBOL } = require('../constants/condition.js');
+
+const SYMBOLS = Object.freeze({
+  start: '[',
+  end: ']',
+  fail: 'X',
+  success: 'O',
+  blank: ' ',
+  separator: ' | ',
+});
+
+class TemplateMaker {
+  static getLogTemplates(logs) {
+    const upsideTemplate = TemplateMaker.#makeLogTemplate(logs, UPSIDE_SYMBOL);
+    const downsideTemplate = TemplateMaker.#makeLogTemplate(logs, DOWNSIDE_SYMBOL);
```

Comment:
음..🤔 뭔가 중복코드인게 아쉬워서 우선 짜보긴했는데, 베스트는 아닐꺼 같네요. 😭

```suggestion
    const [upsideTemplate, downsideTemplate] = [UPSIDE_SYMBOL, DOWNSIDE_SYMBOL].map((symbol)=>{
        TemplateMaker.#makeLogTemplate(logs, symbol);
    });
```

아래 34-35줄도 중복 코드라 조금만 더 고민🤔 을 해보시면 좋은 방법이 있을 것 같아요!
메소드를 하나 더 만들면 방법을 찾으실 수 있을 것 같습니다! 🤓

```js
        if (isCrossable && isSymbolSide) return SYMBOLS.success;
        if (!isCrossable && isSymbolSide) return SYMBOLS.fail;
```